### PR TITLE
Animal table

### PR DESCRIPTION
**Pull request by jellywiz**
# Create Vet-Clinic Database
### in this step of the project I have completed the followings
* Create database
* Create Table
* Insert data
* Perform queries

### Schema
create animal table:
create table

### Data
Inserting data into the database:
insert data

### Queries 
Find all animals whose name ends in "mon".
image

List the name of all animals born between 2016 and 2019.
image

List the name of all animals that are neutered and have less than 3 escape attempts.
image

List the date of birth of all animals named either "Agumon" or "Pikachu".
image

List name and escape attempts of animals that weigh more than 10.5kg
image

Find all animals that are neutered.
image

Find all animals not named Gabumon.
image

Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,18 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon';
+
+SELECT name from animals WHERE date_of_birth BETWEEN '2016-01-01' and '2019-01-01'
+
```
Comment (ShadyShawkat) [inline queries.sql]: - [x] Please make sure to select all animals that have a birth date between the years `2016` and `2019`, meaning that if they had a birth date in the year 2019 (until `2019/31/12`) they should be selected.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @jellywiz, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (jellywiz) [reply queries.sql]: thanks for you help i will make sure to fix that issue

**Review by Munsa1 — APPROVED** [review]
Hi @jellywiz, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jellywiz** [comment]
> Hi @jellywiz,
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉
> 
> ## Highlights
> ✔️ Descriptive Readme ✔️ Passing Linters ✔️ Correct Gitflow
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

thank you for your approval 🔥@Munsa1

**Pull request merged**